src/resources/assets/js/modules/upload.js

```diff
@@ -172,6 +172,14 @@ export default {
                 sending: function (file, xhr, formData) {
                     uploadProgress += parseFloat(100 / allFiles)
                     manager.progressCounter = `${Math.round(uploadProgress)}%`
+                    
+                    //fix if Dropzone doesnt find the form. This only happens if you use direct drag and drop on the div (__stack-container).
+                    if (!formData.has('upload_path')) {
+                        let path = document.getElementById('new-upload').getRootNode().getElementsByName('upload_path')[0].value;
+                        if (path !== '') {
+                            formData.append('upload_path', path);
+                        }
+                    }
```

Comment:
maybe it would be better if we wrap the whole thing inside a try & catch

Reply:
because of the index lookup & what comes after it. as for `getElementById` i prefer to use `querySelector` instead

Reply:
Changed to an `querySelector`